plugins/woocommerce/client/blocks/tests/js/config/testing-library.js

```diff
@@ -2,7 +2,6 @@
  * External dependencies
  */
 require( '@testing-library/jest-dom' );
-require( '@wordpress/jest-console' );
```

Comment:
I mostly wanted this gone as `@wordpress/jest-preset-default` is adding it, so this import is redundant and misleading (I removed it and still saw `console` usage errors which caused a severe head scratching).